components/browser/engine-gecko-beta/src/main/java/mozilla/components/browser/engine/gecko/GeckoEngineSession.kt

```diff
@@ -74,15 +75,17 @@ class GeckoEngineSession(
     @Throws(GeckoEngineException::class)
```

Comment:
Just noticed that we don't throw this anymore. We can also delete that type now.

Reply:
I'll file a follow-up for that.